code/Model/Indexer/Algolia.php

```diff
@@ -13,6 +13,9 @@ class Algolia_Algoliasearch_Model_Indexer_Algolia extends Mage_Index_Model_Index
     /** @var Algolia_Algoliasearch_Helper_Logger */
     protected $logger;
 
+    /** @var Algolia_Algoliasearch_Helper_Indexer */
+    protected $indexer;
```

Comment:
Woul be better to expand the var name to something like `$indexHelper`. Only `$indexer` might be confusing with `Indexer` classes.